src/constants/main.ts

```diff
@@ -45,3 +48,13 @@ export const BOTBUSTERS_LOGO_ANIMATION_SEGMENT = [
   [0, 96],
   [47, 96],
 ];
+
+export const ACHIEVEMENTS_TO_STORE: AchievementId[] = [
```

Comment:
I wonder if at this point we should just create achievements as records, where we store the points and "isStorable" kind of boolean.